Resources/Prototypes/Maps/arena.yml

```
@@ -61,12 +61,14 @@
             Reporter: [ 2, 2 ]
             ServiceWorker: [ 1, 3 ]
           #science
+            AI: [ 1, 1 ]
```

The job slot here is incorrect.

```suggestion
            StationAi: [ 1, 1 ]
```